cmd/main_test.go

```diff
@@ -615,3 +617,117 @@ var _ = DescribeTable("TestGetNamespacesWithAccess", func(allNamespaces []k8sapi
 		})
 	})
 })
+
+var _ = Describe("GetUserNamespaces", func() {
+	var e *echo.Echo
+	var req *labels.Requirement
+	var createdNamespaces []string
+
+	// checks if all created namespaces are in the returned list
+	Context("When querying for all user namespaces using Exists", func() {
+
+		BeforeEach(func() {
+			namesToCreate := []string{"test-ns-1", "test-ns-2", "test-ns-3"}
+			for _, name := range namesToCreate {
+				ns, err := createNamespace(k8sClient, name)
+				Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Error creating the namespace %s", name))
+				createdNamespaces = append(createdNamespaces, ns.Name)
+			}
+
+			var err error
```

Comment:
can't we use `:=` in the following line?

Reply:
@Omeramsc which lines exactly are you talking about?

Reply:
638
```req, err = labels.NewRequirement("kubernetes.io/metadata.name", selection.Exists, []string{})```

Reply:
@Omeramsc When I try to update := on line 638 it throws this error
![image](https://github.com/user-attachments/assets/969adeab-9c6a-47c3-ada0-2bd9e2b4af83)

Reply:
this error applies to `req`, not `err`. debug why it is showing you this error when you do reference req earlier.

Reply:
I think its becasue req is already defined outside the `beforeach` fun. 
:= I see is been used for declaring and initializing new variables. since we have declared req already outside beforeeach we can only use = in here.